src/components/input/input.e2e.ts

```diff
@@ -0,0 +1,43 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+it(`Propagates wcsChange event when a key stroke occurs on the native input`, async () => {
+    // Given
+    const page = await newE2EPage({
+        html: `
+                <wcs-input />
+            `
+    });
+    const select = await page.find('wcs-input');
+    const input_wrapper = await page.find('wcs-input >>> input');
+    const changeSpy = await select.spyOnEvent('wcsChange');
+
+    // When
+    await input_wrapper.press('A');
+    await page.waitForChanges();
+
+    // Then
+    expect(changeSpy).toHaveReceivedEventTimes(1);
+    console.dir(changeSpy.firstEvent.detail);
+    expect(changeSpy).toHaveReceivedEventDetail({ value: 'A' });
+});
+
+xit(`Propagates wcsInput event when a key stroke occurs on the native input`, async () => {
+    // Given
+    const page = await newE2EPage({
+        html: `
+                <wcs-input />
```

Comment:
Custom elements are not intended to be used as self-closing, browser behavior is UB on this.